### ISSUE TITLE
Deprecating Ontology managers

### ISSUE THREAD

**Issue by LckyLke** [dice-group/owlapy#124]
As talk about in our meeting on Monday, we do not use ontology managers to apply changes to ontologies at the moment. 
In their current implementation they don't really serve any obvious purpose. 

In the OWL API library the ontology managers are used to centralize management, i.e. creation, loading, saving, and modification of ontologies. We could either do it this way or apply changes directly on the Ontologies, which would eliminate the need for ontology managers.

**Comment by LckyLke** [comment]
"All ontology changes are applied through an ontology manager. This means that it is possible to apply a list of ontology changes, which make multiple changes to multiple ontologies, as a single unit. This works well for applications such as ontology editors, where an edit operation such as entity name change (entity IRI change) can involve the addition and removal of multiple axioms from multiple ontologies..." from [here](https://www.semantic-web-journal.net/sites/default/files/swj107.pdf).

If we also want to support something like this, managers might be useful

**Comment by Demirrr** [comment]
@alkidbaci agreed ?

**Comment by alkidbaci** [comment]
Yes I think we can depreciate the ontology manager because as Luke said, they dont really serve any purpose now. Should keep in mind during refactoring that "Sync" classes still need to use owlapi's OntologyManager. 

Also `OWLImportsDeclaration` and `AddImports` should be depricated along with the manager.
